liquidhaskell.cabal

```diff
@@ -135,7 +135,7 @@ Library
                 , vector >= 0.10
                 , hashable >= 1.2
                 , unordered-containers >= 0.2
-                , liquid-fixpoint >= 0.5 && < 0.6
+                , liquid-fixpoint 
```

Comment:
Please leave this bound in place, liquid-fixpoint changes API so much it's pretty much guaranteed the next hackage release won't work with current LH.